### PR TITLE
TFKUBE-226: Add test for remote agents

### PR DESCRIPTION
**Pull request by nghazali**

### CODE REVIEW

**Comment by jjeongatl** [comment]
Can you remove line 4-5 in `installer_test.go`? Go linter complains because they are unused imports

**Comment by nghazali** [comment]
Thanks for testing. Let's clean up the code in TFKUBE-48 to avoid breaking the code. 

This maybe needed for TFKUBE-224 and TFKUBE-225 - the reason is we implemented all together, and split the relevant code to raise different PRs.

File: test/e2etest/installer_test.go [hunk 1]
```diff
@@ -27,10 +27,19 @@ func TestInstaller(t *testing.T) {
...
+	assertRemoteAgentList(t, testConfig)
+
 	// Uninstall and cleanup the environment
 	runUninstallScript(testConfig.ConfigPath)
 }
 
+func assertRemoteAgentList(t *testing.T, testConfig TestConfig) {
+	agentUrl := "admin/agent/configureAgents!doDefault.action"
+	url := fmt.Sprintf("https://%s@%s.%s.%s/%s", credential, product, testConfig.EnvironmentName, domain, agentUrl)
+	content := fmt.Sprintf("%s", GetPageContent(t, url))
```
Comment (yzha645) [inline test/e2etest/installer_test.go:40]: Same as https://github.com/atlassian-labs/data-center-terraform/pull/103#discussion_r786342371

File: test/e2etest/installer_test.go [hunk 2]
```diff
@@ -27,10 +27,19 @@ func TestInstaller(t *testing.T) {
...
+	assert.Contains(t,content, "There are currently 3 remote agents online.")
```
Comment (yzha645) [inline test/e2etest/installer_test.go:41]:
```suggestion
	assert.Contains(t, content, "There are currently 3 remote agents online.")
```

File: test/e2etest/installer_test.go (hunk 1 above)
Comment (nghazali) [reply test/e2etest/installer_test.go:40]: replied in #103

**Review by yzha645 — APPROVED** [review]

**Pull request merged**